src/AutoRest.CSharp/Common/Output/PostProcessing/PostProcessor.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoRest.CSharp.AutoRest.Plugins;
+using AutoRest.CSharp.Mgmt.Output;
+using AutoRest.CSharp.Output.Models.Types;
+using AutoRest.CSharp.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Simplification;
+
+namespace AutoRest.CSharp.Common.Output.PostProcessing;
+
+internal abstract class PostProcessor
```

Comment:
I think the public methods of PostProcessor can be tested in unit tests.
Could you add some tests to verify the functionality?

Reply:
It is testable, but first I would like to focus on the feature

Reply:
Please consider this work in the MQ